File: Data Structures & Algorithms/subarray-sum-equals-k/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number}
     */
    subarraySum(nums, k) {
        let count = 0;
        let n = nums.length;
        for(let i=0;i<n;i++) {
            let windowSum = 0;
            for(let j=i;j<n;j++) {
                windowSum = windowSum + nums[j];
                if(windowSum === k) {
                    count++;
                }
            }
        }
        return count;
    }
}

File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int xor = 0;
        for(int i=0;i<n+1;i++) {
            xor = xor ^ i;
        }
        for(int i : nums) {
            xor = xor ^ i;
        }
        return xor;
    }
}
